workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs

```diff
@@ -21,6 +23,25 @@ protected ComponentVisualElement()
             AuthoritativeToggle = this.Q<Toggle>(className: "is-auth-toggle");
         }
 
+        protected void InjectComponentIcon(string iconName)
+        {
+            var iconContent = EditorGUIUtility.IconContent(iconName);
+            var iconVisualElement = new VisualElement();
+            iconVisualElement.style.backgroundImage = new StyleBackground((Texture2D) iconContent.image);
+            iconVisualElement.AddToClassList("component-icon");
+
+            // We want to inject the icon after the toggle icon and before the text.
+            var foldoutToggle = ComponentFoldout.Q<VisualElement>(className: "unity-toggle__input");
+            var children = foldoutToggle.Children().ToList();
```

Comment:
This looks a little hacky, could be made simpler using `PlaceBehind`?

Reply:
Turns out there's an `Insert(index, element)` method which I just... missed.. the first time around!